src/DetailsView/components/assessment-instance-edit-and-remove-control.tsx

```diff
@@ -12,10 +12,14 @@ import { CapturedInstanceActionType, FailureInstancePanelControl } from './failu
 export interface AssessmentInstanceEditAndRemoveControlProps {
     test: VisualizationType;
     step: string;
-    id: string;
-    description: string;
+    currentInstance: {
```

Comment:
You can extract this out to a type of it's own and use that type everywhere instead.

Reply:
Abstracted to its own type (Failure Instance Data) and now propagates that down in props to create the Failure Instance Panel Control! In a separate PR I will simplify the onAddFailureInstance and onEditFailureInstance pathways to use this data structure as well--just wanted to keep this PR as simple as possible for the time being. Thanks!

Reply:
Have extracted out to type and propagated up so this type is used in the creation of the panel itself AND propagated down so its used in the store changes prompted by AddFailureInstance and EditFailureInstance